english/33hours/pillow.md

```diff
@@ -0,0 +1,3 @@
+As you lift the pillow to see what is underneath, the strong smell of dried blood caking on top nauseates you. You feel dizzy again. But you sense something metallic under the pillow. You want to check it out, but your eyes are closing. It is getting darker. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.